packages/lesswrong/components/posts/PostsPage/PostsPageEventData.tsx

```diff
@@ -58,6 +90,36 @@ const PostsPageEventData = ({classes, post}: {
     </a> : <div>Online Event</div>
   }
   
+  // if this event was labelled with an event type, display it
+  let eventTypeNode;
+  switch (eventType) {
+    case 'presentation':
+      eventTypeNode = <div className={classes.eventType}>
+        <WebIcon className={classes.eventTypeIcon} /> PRESENTATION
+      </div>
```

Comment:
My first instinct was to use something like [MUI chips](https://mui.com/components/chips/) to hold the filterable event metadata (then we could make clicking on them link to the list of events with that filter already applied). But I think the forum is more minimalist than I am used to, so I decided to just keep it a very light grey icon + text, at least for now.

Reply:
It took me a minute to notice the event type even though I was specifically looking for it.  Maybe it should be a little darker? I agree with not having a chip though

Reply:
Hmm yeah I do think it should be relatively unnoticeable, since it's less important than the rest of the info. But I still wanted it to feel separate from the rest, so I went with adding a bit of orange:

<img width="635" alt="Screen Shot 2022-02-04 at 11 32 34 AM" src="https://user-images.githubusercontent.com/9057804/152522277-5f7b5b2d-adbd-47a3-bbac-0fc40825628e.png">